pkg/lib/operatorstatus/csv_reporter.go

```diff
@@ -76,7 +76,7 @@ func (r *csvStatusReporter) GetNewStatus(existing *configv1.ClusterOperatorStatu
 		gvk := csv.GetObjectKind().GroupVersionKind()
 
 		builder.WithoutVersion(csv.GetName(), csv.Spec.Version.String()).
-			WithoutRelatedObject(gvk.Group, gvk.Kind, csv.GetNamespace(), csv.GetName())
+			WithoutRelatedObject(gvk.Group, "clusterserviceversions", csv.GetNamespace(), csv.GetName())
```

Comment:
It would be nice if we could thread the resource through here the same way we do the GVK from context.

For now, just making this string a constant (since the CSV-reported is pretty specific to CSVs anyway) would help.

Reply:
Added in a constant for this